packages/@react-aria/selection/test/useSelectableCollection.test.js

```diff
@@ -52,4 +55,27 @@ describe('useSelectableCollection', () => {
     expect(options[1]).not.toHaveAttribute('aria-selected');
     expect(options[2]).toHaveAttribute('aria-selected', 'true');
   });
-});
\ No newline at end of file
+
+  it('uses toggle mode in mobile', () => {
```

Comment:
is it better to do it by mobile? or if we detect that it's touch driven interaction? what if they plug a keyboard into their phone?

Reply:
Prob better to detect that it is touch driven interaction IMO, don't think iPads and tablets get caught by `useIsMobileDevice` (need to double check this, gotta charge my test ipad).

Reply:
after thinking about it some, driving this as a decision of touch does seem better
I think a touch user wouldn't reach for a keyboard modifier key